Sources/SwipeCell/SwipeCellViewModifier2.swift

```diff
@@ -58,9 +58,18 @@ extension SwipeCellModifier {
         .contentShape(Rectangle())
         .gesture(getGesture())
         .onAppear {
-            withAnimation(Animation.default.delay(initialOffsetResetDelay)) {
-                if resetOffsetOnAppear {
-                    offset = 0
+            self.setStatus(status)
+            switch status {
+            case .showLeftSlot:
+                offset = leftSlotWidth
+            case .showRightSlot:
+                offset = rightSlotWidth
+            default:
+                break
+            }
+            DispatchQueue.main.asyncAfter(deadline: .now() + initialStatusResetDelay) {
```

Comment:
Not sure if there is a better way to do this with a timer or something. Open to suggestions

Reply:
I should do the same